packages/dids/src/methods/did-dht.ts

```diff
@@ -1168,14 +1196,20 @@ export class DidDhtDocument {
       // Use the public key's `crv` property to get the DID DHT key type.
       const keyType = DidDhtRegisteredKeyType[publicKey.crv as keyof typeof DidDhtRegisteredKeyType];
 
+      let alg;
+      if(KeyTypeToDefaultAlgorithmMap[keyType] !== getJoseSignatureAlgorithmFromPublicKey(publicKey))
+      {
+        alg = getJoseSignatureAlgorithmFromPublicKey(publicKey);
+      }
+
       // Convert the public key from JWK format to a byte array.
       const publicKeyBytes = await DidDhtUtils.keyConverter(publicKey.crv).publicKeyToBytes({ publicKey });
 
       // Convert the public key from a byte array to Base64URL format.
       const publicKeyBase64Url = Convert.uint8Array(publicKeyBytes).toBase64Url();
 
       // Define the data for the DNS TXT record.
-      const txtData = [`id=${methodId}`, `t=${keyType}`, `k=${publicKeyBase64Url}`];
+      const txtData = [`id=${methodId}`, `t=${keyType}`, `k=${publicKeyBase64Url}`, ...(alg ? [`alg=${alg}`] : [])];
```

Comment:
```suggestion
      const txtData = [`id=${methodId}`, `t=${keyType}`, `k=${publicKeyBase64Url}`, ...(alg ? [`a=${alg}`] : [])];
```

https://did-dht.com/#verification-methods